README.md

```diff
@@ -1,256 +1,304 @@
-## Arch Linux CN Community repo mirrors list
+# Arch Linux CN Community Repository Mirror List
 
 Here is a list of public mirrors of our [community repository](https://github.com/archlinuxcn/repo).
 
-If you interested in making a mirror of our repository, please open an issue or pull request (or contact us at repo@archlinuxcn.org and hope the mail reaches).
+Locate the nearest mirror to you in [map](https://archlinuxcn.org/mirrors/map.html), and monitor the synchronization status of mirrors on [dashboard](https://build.archlinuxcn.org/grafana/d/iK2vLpGGk/mirrors).
+
+## Apply Mirror
+
+If you are interested in applying mirror of our repository, please refer to the [application.md](application.md) for instructions.
+
+## Debuginfod Configuration
+
+(This is included in `archlinuxcn-mirrorlist` package.)
+
+```bash
+cp -v archlinuxcn.urls /etc/debuginfod/
+```
+
+## Mirror List
+
+(You can also view a more beautiful mirror list in [here](https://archlinuxcn.org/mirrors/list.html).)
+
+### Our main server
 
 ```ini
-## Our main server (Amsterdam, the Netherlands) (ipv4, ipv6, http, https)
+## Our main server (Amsterdam, North Holland, Netherlands) (http, https, ipv4, ipv6)
 [archlinuxcn]
 Server = https://repo.archlinuxcn.org/$arch
 ```
 
+### 校园网联合镜像站
+
 ```ini
-## CERNET (中国) (ipv4, ipv6, http, https)
-## Added: 2023-08-19
-## This will redirect you to the nearest China educational mirror site. However, please note that some sites may not support IPv6.
+## 校园网联合镜像站 (中国) (ipv4, ipv6)
+## Redirect to suitable educational mirror based on location
 [archlinuxcn]
 Server = https://mirrors.cernet.edu.cn/archlinuxcn/$arch
 ```
```

Comment:
This is an automatically redirected mirror site, so it is impossible to get a status code of 200 as a response.